src/_tutorials/server/get-started.md

```diff
@@ -1,263 +1,237 @@
 ---
-title: Get started with server-side Dart
-description: Get Dart and run a Dart app.
+title: "Get started: Dart native apps"
+description: Get Dart, run and compile a small app
+toc: true
 nextpage:
   url: /tutorials/server/cmdline
   title: Write command-line apps
 prevpage:
   url: /tutorials/server
-  title: Server-side Dart tutorials
+  title: Tutorials
```

Comment:
I'm not sure about this nav change.

Reply:
This link is the the tutorial that is in this PR. That is a general tutorial that teaches the tools used for all command-line & server apps, so having the title just say 'server-side' is wrong.

Reply:
Right, but it's not the top-level tutorial page, either. (e.g. it doesn't cover web apps & Flutter)

Reply:
Changed to 'Dart command-line and server tutorials'